staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -108,6 +108,7 @@ func PatchResource(r rest.Patcher, scope RequestScope, admit admission.Interface
 		userInfo, _ := request.UserFrom(ctx)
 		staticAdmissionAttributes := admission.NewAttributesRecord(nil, nil, scope.Kind, namespace, name, scope.Resource, scope.Subresource, admission.Update, userInfo)
 		admissionCheck := func(updatedObject runtime.Object, currentObject runtime.Object) error {
+			// if we allow this some day, we have this TODO: call the mutating admission chain with the CREATE verb instead of UPDATE
```

Comment:
what is "this"? 🙂 needs some context, like "if we allow create-on-patch, we have this TODO: